webapp/src/containers/SettingsPage/saga.tsx

```diff
@@ -207,7 +211,9 @@ export function* setLockTimeout(action) {
     const timeout = action.payload;
     const resp = yield call(updateLockTimeout, timeout);
     if (resp?.success) {
-      log.info(`Timeout updated to ${timeout} seconds`, 'updatePassphrase');
```

Comment:
Can we add translations for this?

Reply:
Yes I have added translations instead of hard coded value